cmd/restore.go

```diff
@@ -26,7 +26,7 @@ import (
 	"syscall"
 	"time"
 
-	"gx/ipfs/QmXLwxifxwfc2bAwq6rdjbYqAsGzWsDE9RM5TWMGtykyj6/interface-go-ipfs-core"
+	iface "gx/ipfs/QmXLwxifxwfc2bAwq6rdjbYqAsGzWsDE9RM5TWMGtykyj6/interface-go-ipfs-core"
```

Comment:
why do we need aliases on all these?

Reply:
They are there for convenience and added by `goimports` on saving the file. When the package has a conflicting name with another package or doesn't match the final token of the import path, putting the name helps the dev easily identify which package provides `iface` in the code below.

Reply:
I know what it does ;) I'm just saying previously you've been hesitant to add those changes when not absolutely necessary like on my last PR. So was just curious why this time was different.

Reply:
I'm not sure if this was the case on your PR where I commented, but if it's just the imports being changed and nothing else in the file, I'd rather the change be omitted as it's functionally a no-op change. Of course, it's also possible I'm just being inconsistent. We generally have a lot noise in our repo and that's something that I grapple back and forth with being pedantic about and just accepting. (Of course, a lot of this must be accepted due to golang's opinionated formatting.)

Reply:
I agree with Brian but not enough to hold up this PR I guess.

Reply:
I don't mind removing them. We previously agreed to remove the goimports requirement from the linter to revert back to gofmt and my editor is still configured to run goimports on save. I've removed that settings to use gofmt again so this doesn't keep happening.